### PR TITLE
Initialize the file/folder structure for Volibri

### PR DESCRIPTION
**Pull request by 66eli77**
the start point for building up the version controlled UI/UX design

### CODE REVIEW

**Comment by 66eli77** [comment]
@whitzhu @jtamiace  please comment on any confusing part or confusing terminology in the README file

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+**The problems:**
+
+It appears to us that a version control solution is needed to accommodate the collaborative design work we've done for Kolibri.
+There are 3 major issues we want to address with this version control solution:
+
+1. Be able to easily trace back the design decision making history. Following describes a typical issue we are facing more and more often. After many iterations for an UI element, it becomes unclear to us how we arrived at the current approach, and start bringing arguments up that later we realized were fully discussed before.
+
+2. Be able to keep design modifications in sync across the team. Our design team has collectively experienced this pain that if one of us updates the design of a shared UI element, all of us have to find the files that uses this shared element and manually update it.
+
+3. Be able to asynchronously comment on and reply to new designs. We used to post designs in our online chat room to get feedbacks. But just like design takes inspiration, criticizing design also takes inspiration. Since we cannot guarantee to get inspired in time, let's do it asynchronously via Pull Request.
```
Comment (jtamiace) [inline README.md]: Alternative way of framing this? Was unclear at first. 
From discussion: provide example, of design discussion being pushed up and becoming irrelevant in the Slack #design room

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+
+**The solution:**
+
+Terminology: `link-import` means using `Place-Linked-Bitmap` plugin's `Place Bitmap as New Layer` method to create a linkage of the external image file. Therefore, whenever the external image files changed, we can use the `Update All Bitmaps` method to update all link-imported images in the Sketch file.
```
Comment (jtamiace) [inline README.md]: For those who are new, might be good to further define what linkage means?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,57 @@
...
+There is an example how to build a list UI element uisng this approach.
+
+1. We create 2 Sketch files, `container_element.sketch` and `list_item_element.sketch`.
+
+2. We use the `Make Exportable` feature of Sketch to generate 2 image files, `container.png` and `list-item.png`. (Note: because we installed `sketch-image-compressor` plugin, all exported images will be automatically losslessly compressed.)
+
+3. We create another Sketch file `list.sketch` that `link-import` the `container.png` and `list-item.png`, place the imported list-item inside the imported container, copy paste(the duplications will keep the linkage to the external image) the list-item couple times to make it looks like a proper list UI element.
+
+Say later you want to change the background color of the list-item, you can open the `list_item_element.sketch` and change the background color. Export image again to update the `list-item.png` with your modification. (it is important to keep the image file name unchanged!). Open your `list.sketch`, click `Plugins/Place Linked Bitmap/Update All Bitmap`, you will see that all the list-items' background color get changed to the new color.
```
Comment (jtamiace) [inline README.md]: Could make it easier to read by presenting these steps in list format?

**Pull request merged**